aemanalyser-core/src/main/java/com/adobe/aem/analyser/Runmode.java

```diff
@@ -0,0 +1,26 @@
+/*************************************************************************
+ * ADOBE CONFIDENTIAL
+ * __________________
+ *
+ *  Copyright 2022 Adobe
+ *  All Rights Reserved.
+ *
+ * NOTICE: All information contained herein is, and remains
```

Comment:
The repo claims that this code is supposed to be licensed under ASL 2.0, this header does not indicate that.

Reply:
Whoops thanks, @kwin ! That's wrong. I'll fix that.